ios/FluentUI/ActivityIndicator/ActivityIndicatorTokens.swift

```diff
@@ -16,40 +16,22 @@ import UIKit
 }
 
 /// Design token set for the `ActivityIndicator` control.
-public class ActivityIndicatorTokens: ControlTokens {
-    /// Creates an instance of `ActivityIndicatorTokens` with optional token value overrides.
-    /// - Parameters:
-    ///   - size: MSFActivityIndicatorSize enumeration value that will define pre-defined values for side and thickness.
-    ///   - defaultColor: The default color of the Activity Indicator.
-    ///   - side: The value for the side of the square frame of an Activity Indicator.
-    ///   - thickness: The value for the thickness of the ActivityIndicator ring.
-    public init(size: MSFActivityIndicatorSize,
-                defaultColor: DynamicColor? = nil,
-                side: CGFloat? = nil,
-                thickness: CGFloat? = nil) {
-
-        self.size = size
-        super.init()
-
-        // Optional overrides
-        if let defaultColor = defaultColor {
-            self.defaultColor = defaultColor
-        }
-        if let side = side {
-            self.side = side
-        }
-        if let thickness = thickness {
-            self.thickness = thickness
-        }
+open class ActivityIndicatorTokens: ControlTokens {
+    /// MSFActivityIndicatorSize enumeration value that will define pre-defined values for side and thickness.
+    public var size: MSFActivityIndicatorSize {
+        guard let state = state else { preconditionFailure() }
```

Comment:
This seems like a very odd way to treat an optional variable. I think I understand that state is weak to try and avoid retention cycles, but it doesn't seem like it should be a weak variable if we're going to try and crash when the value isn't there?

Reply:
Maybe we could change one of the references to unowned? Seems like an unowned reference in the state impl should also work.

Reply:
Ah hm I think I can make that work using a `!` reference. Thanks for the idea!